scripts/helper_functions.sh

```diff
@@ -2,8 +2,12 @@
 
 # Common functions definitions
 
+function get_php_version {
+    local PHPVER=`/usr/bin/php -r "echo PHP_VERSION;" | /usr/bin/cut -c 1,2,3`
+    echo ${PHPVER}
```

Comment:
This is OK, but it's going to run the external commands every time this function gets called. Probably we can save the result to a global (I know we want to avoid this, but for the sake of some perf improvement...) and return it if it's populated? Something like:

```
function get_php_version {
    if [ -z "$_PHPVER" ]; then
        _PHPVER=`/usr/bin/php -r "echo PHP_VERSION;" | /usr/bin/cut -c 1,2,3`
    fi
    echo $_PHPVER
}
```